sdk/src/main/kotlin/au/kinde/sdk/KindeSDK.kt

```
@@ -347,11 +350,25 @@ class KindeSDK(
+                    else -> return false
+                }
+                
+                val expireEpochMillis = expirySeconds * 1000
+                val currentTimeMillis = System.currentTimeMillis()
 
-            if (currentTimeMillis > expireEpochMillis) {
-                return true
+                if (currentTimeMillis > expireEpochMillis) {
+                    return true
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)